reference/promotion_settings.v3.yml

```diff
@@ -0,0 +1,316 @@
+            schema:
+              $ref: "#/components/schemas/GlobalPromotionSettings"
+            example:
+              promotions_triggered_by_products_with_zero_product_price: true
+              promotions_apply_on_products_with_custom_product_price: false
+              number_of_coupons_allowed_at_checkout: 1
+              promotions_applied_on_original_product_price: true
+        required: true
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GlobalPromotionSettings"
+              example:
+                promotions_triggered_by_products_with_zero_product_price: true
+                promotions_apply_on_products_with_custom_product_price: false
+                number_of_coupons_allowed_at_checkout: 1
+                promotions_applied_on_original_product_price: true
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse400"
+        "422":
+          description: "Promotion settings data is not valid. This is the result of missing required fields, or of invalid data. See the response for more details."
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse422"
+
+components:
+  schemas:
+    PromotionSettingsResponse:
+      type: object
+      required:
+        - data
+      properties:
+        data:
+          $ref: "#/components/schemas/GlobalPromotionSettings"
+    GlobalPromotionSettings:
+      type: object
+      required:
+        - promotions_apply_on_products_with_custom_product_price
+        - promotions_triggered_by_products_with_zero_product_price
+        - number_of_coupons_allowed_at_checkout
+        - promotions_applied_on_original_product_price
+      properties:
+        promotions_apply_on_products_with_custom_product_price:
+          type: boolean
+          example: false
+          default: false
+          description: |-
+            - When the property is set to "true", promotions will apply for products with custom pricing.
+            - When the property is set to "false", promotions will not apply for products with custom pricing.
+            - Note: This field can only be edited for Enterprise Accounts.
+        promotions_triggered_by_products_with_zero_product_price:
+          type: boolean
+          example: false
+          default: false
+          description: |-
+            - When the property is set to "true", promotions will be triggered by zero price products.
+            - When the property is set to "false", promotions will not be triggered by zero price products.
+            - Note: This field can only be edited for Enterprise Accounts.
+        number_of_coupons_allowed_at_checkout:
+          minimum: 1
+          maximum: 5
```

Comment:
@jpajar would you please update this to 15 as per https://github.com/bigcommerce/bigcommerce/pull/58981#discussion_r1687266855 ? thanks.